### ISSUE TITLE
Write training script for arg-passing first task (KAE, constrained KAE, regular AE)

### ISSUE THREAD

**Issue by charlieoneill11** [jackmiller2003/kae-cyclones#2]
At the moment, training is done in notebooks. Would be good to set up a script to run training from command line with arguments for models, etc. Also output the loss for all functions, and combine W&B logging.

**Comment by charlieoneill11** [comment]
Need to add argument for choosing model
